app/templates/components/dialogs/cancel-subscription-modal.hbs

```diff
@@ -1,21 +1,11 @@
-{{#if @isOpen}}
-  <ModalDialog
-    @targetAttachment="center"
-    @translucentOverlay={{true}}
-    @overlayPosition="sibling"
-    @clickOutsideToClose={{true}}
-    @animatable={{true}}
+
```

Comment:
I thought it might be nice if Modal showed and hid itself, to reduce the logic necessary to use it. It also made it a little easier to implement the animation. As a result right now a few of these templates are not properly indented, or have a superfluous div to make it more obvious what actually changed, to make at least the initial review a little easier. I would like to properly indent these before merging (or if people feel it would be better to keep outside-if logic, revert to that)

Reply:
This is a great idea, @sorta, thank you for your care, it's so much appreciated ❤️